stile/sys_tests.py

```diff
@@ -1447,3 +1447,26 @@ def __call__(self, array, per_ccd_stat = None, color = '', lim=None):
                                                                        linear_regression=True,
                                                                        reference_line='zero')
 
+
+class ScatterPlotResidualSigmaVsPSFMagSysTest(ScatterPlotSysTest):
+    short_name = 'scatterplot_residual_sigma_vs_psf_magnitude'
+    long_name = 'Make a scatter plot of residual sigma vs PSF magnitude'
+    objects_list = ['star PSF']
+    required_quantities = [('sigma', 'sigma_err', 'psf_sigma', 'mag_inst')]
+
+    def __call__(self, array, per_ccd_stat = 'None', color = '', lim=None):
+        self.per_ccd_stat = None if per_ccd_stat == 'None' else per_ccd_stat
+        import numpy.lib.recfunctions
+        use_array = numpy.copy(array)
+        use_array = numpy.lib.recfunctions.append_fields(use_array, 'sigma_residual_frac', (use_array['sigma'] - use_array['psf_sigma'])/use_array['psf_sigma'])
+        use_array = numpy.lib.recfunctions.append_fields(use_array, 'sigma_residual_frac_err', use_array['sigma_err']/use_array['psf_sigma'])
+        return super(ScatterPlotResidualSigmaVsPSFMagSysTest, self).__call__(use_array, 'mag_inst', 'sigma_residual_frac', 'sigma_residual_frac_err',
```

Comment:
Line length problems here too.  Actually there are line length problems for all the `ScatterPlot` tests, which I hadn't noticed before--if you'd like to make that a separate issue (a general formatting/spacing/etc issue would also be fine) we could fix it there?  Or just do it here, I guess.